Lib/sqlite3/test/regression.py

```diff
@@ -376,6 +376,37 @@ def CheckCommitCursorReset(self):
                 counter += 1
         self.assertEqual(counter, 3, "should have returned exactly three rows")
 
+    def CheckUninitializedIsolationLevel(self):
+        """
+        Previously trying to get the isolation level of an uninitialized Connection
+        caused a segfault, now it should just return None.
```

Comment:
Other getters raise a `ProgrammingError` when they are called on an uninitialized `Connection` object.
IMHO this should be the same here.